web/yo/app/scripts/controllers/gene.js

```diff
@@ -105,6 +105,21 @@ angular.module('oncokbApp')
                     console.log('Unable to add comment.');
                 }
             };
+            $scope.vusUpdate = function(message) {
+                if ($scope.status.isDesiredGene) {
+                    var vus = $scope.realtimeDocument.getModel().getRoot().get('vus');
+                    var vusData = stringUtils.getVUSFullData(vus, true);
+                    DatabaseConnector.updateVUS($scope.gene.name, JSON.stringify(vusData), function(result) {
+                        console.log('success saving vus to database');
+                    }, function(error) {
+                        console.log('error happened when saving VUS to DB', error);
+                        var subject = 'VUS update Error for ' + $scope.gene.name.getText();
+                        var content = 'Error happened when ' + message + '. The system error returned is ' + error;
```

Comment:
@JiaoJiao123 one last thing, could you also include who modified the content and the date? Thanks.